etc/win-ci/crystal.iss

```diff
@@ -0,0 +1,335 @@
+#define MyAppName "Crystal"
+#define VersionFile FileOpen("portable\src\VERSION")
+#define MyAppVersion FileRead(VersionFile)
+#define MyAppVersionNum StringChange(MyAppVersion, "-dev", "")
+#expr FileClose(VersionFile)
+#define MyAppPublisher "Manas Technology Solutions"
+#define MyAppURL "https://crystal-lang.org/"
+#define MyAppExeName "crystal.exe"
+#define MyAppAssocName MyAppName + " Source File"
+#define MyAppAssocExt ".cr"
+#define MyAppAssocKey StringChange(MyAppAssocName, " ", "") + MyAppAssocExt
+
+[Setup]
+AppId={{7C307DDF-447E-46C5-BB3B-47A6F652D7C8}
+AppName={#MyAppName} x86_64-windows-msvc

```

Comment:
Not sure if it makes sense to put the target triple in the app name.

Reply:
The intent is to distinguish between MSVC and any possible MinGW-based toolchains in the future, allowing both to be installed side-by-side. Indeed the Windows part is redundant in this case. For the architecture part there will be ARM64 one day and it is customary to have `(x64)` or `(64-bit)` somewhere in the application name.

`crystal.exe` itself also has the target triple in its manifest. I think using the triple directly is the easiest for tooling purposes.